addons/advanced_throwing/functions/fnc_drawArc.sqf

```diff
@@ -66,7 +66,7 @@ for "_i" from 0.05 to 1.45 step 0.1 do {
 
         private _col = [ [1, 1, 1, _alpha], [0, 1, 0, _alpha], [1, 0, 0, _alpha], [1, 1, 0, _alpha] ] select _cross;
 
-        if (_cross != 2 && {lineIntersects [eyePos ACE_player, _newTrajASL]}) then {
+        if (_cross != 2 && {!((lineIntersectsSurfaces [eyePos ACE_player, _newTrajASL]) isEqualTo [])}) then {
```

Comment:
this seems to intersect with player a lot, not sure why old code didn't
test
```
private _a = lineIntersects [eyePos ACE_player, _newTrajASL];
private _b = !((lineIntersectsSurfaces [eyePos ACE_player, _newTrajASL]) isEqualTo []);
private _c = (lineIntersectsSurfaces [eyePos ACE_player, _newTrajASL]);
systemChat format ["%1 - %2 - %3", _a, _b, _c];
```
[false, true, ... player...]

Reply:
Reverted